### PR TITLE
feat:Refactored the UI for about page

### PR DESCRIPTION
**Pull request by JoeSteven**
refactored the ui for about page according to the new ui design

### CODE REVIEW

File: app/src/main/kotlin/com/twidere/twiderex/component/foundation/ParallaxLayout.kt [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+    backContentOffsetX: Dp = 0.dp,
+    backContentOffsetY: Dp = 0.dp,
+    animateDuration: Int = 300,
+    backContent: @Composable () -> Unit,
+    frontContent: @Composable () -> Unit,
+) {
+    var rotateHorizontal by remember { mutableStateOf(0f) }
+    var rotateVertical by remember { mutableStateOf(0f) }
+    var transitionHorizontal by remember { mutableStateOf(0f) }
+    var transitionVertical by remember { mutableStateOf(0f) }
```
Comment (Tlaster) [inline app/src/main/kotlin/com/twidere/twiderex/component/foundation/ParallaxLayout.kt:62]: Define a `ParallaxLayoutState` and mark it as `@Stable` class, store these states in the `ParallaxLayoutState`, use it with a custom saver

File: app/src/main/kotlin/com/twidere/twiderex/scenes/settings/AboutScene.kt [hunk 2]
```diff
@@ -77,39 +91,105 @@ private fun AboutContent() {
...
+            val parallaxPaddingTop = standardPadding * 5
+            val parallaxPaddingBottom = standardPadding * 10
+            val grayLogoPainter = painterResource(id = R.drawable.ic_about_gray_logo)
+            val aspectRatio = grayLogoPainter.intrinsicSize.width / grayLogoPainter.intrinsicSize.height
+
+            ParallaxLayout(
+                modifier = Modifier
+                    .fillMaxHeight()
+                    .align(Alignment.TopEnd)
+                    .horizontalScroll(state = ScrollState(0), enabled = false),
```
Comment (Tlaster) [inline app/src/main/kotlin/com/twidere/twiderex/scenes/settings/AboutScene.kt:112]: Use `requiredWidth(IntrinsicSize.Max)` for proper usage

**Review by Tlaster — CHANGES_REQUESTED** [review]

File: app/src/main/kotlin/com/twidere/twiderex/component/foundation/ParallaxLayout.kt (hunk 1 above)
Comment (JoeSteven) [reply app/src/main/kotlin/com/twidere/twiderex/component/foundation/ParallaxLayout.kt]: Updated

File: app/src/main/kotlin/com/twidere/twiderex/scenes/settings/AboutScene.kt (hunk 2 above)
Comment (JoeSteven) [reply app/src/main/kotlin/com/twidere/twiderex/scenes/settings/AboutScene.kt:112]: requiredWidth(IntrinsicSize.Max) is not suitable for this ui case ,  i tried, not working

**Review by Tlaster — APPROVED** [review]
LGTM!

**Pull request merged**